Firestore/core/test/unit/local/local_serializer_test.cc

```diff
@@ -286,7 +286,10 @@ class LocalSerializerTest : public ::testing::Test {
         serializer.DecodeTargetData(&reader, *message);
 
     EXPECT_OK(reader.status());
-    EXPECT_EQ(target_data, actual_target_data);
+    // Set the expected_count in expected TargetData to null,  as serializing
```

Comment:
ultranit: In "null, as serializing", replace the two spaces with just one space.